cg/meta/rsync/rsync_api.py

```diff
@@ -140,10 +156,60 @@ def create_log_dir(self, dry_run: bool) -> None:
         else:
             log_dir.mkdir(parents=True, exist_ok=True)
 
+    def get_folders_to_deliver(
+        self, case_id: str, sample_files_present: bool, case_files_present: bool
+    ) -> List[str]:
+        """Returns a list of all the folder names depending if sample and/or case data is to be
+        transferred"""
+        if not sample_files_present and not case_files_present:
+            LOG.error("Since no file parameter is true, no files will be transferred")
+            raise CgError()
```

Comment:
```suggestion
            raise CgError("Since no file parameter is true, no files will be transferred")
```
Nice to have a message if something calls this method and wants to display or log the error